checkin.py

```
@@ -604,6 +807,39 @@ def main():
+        send_email_notification(checkin_results, execution_time)
+    elif os.environ.get('EMAIL_SMTP_HOST'):
+        print('[警告] 已配置邮件参数但无法导入通知模块')
+
+    # 发送 ServerChan 通知
+    if send_serverchan_notification:
+        print('正在发送 ServerChan 通知...')
+        send_serverchan_notification(checkin_results, execution_time)
+    elif os.environ.get('SERVERCHAN_SENDKEY'):
+        print('[警告] 已配置 SERVERCHAN_SENDKEY 但无法导入通知模块')
+
+    # 回写 .env（仅限从 .env 加载且 session 有更新的本地运行）
```

Ruff (0.15.20)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)